src/logic/calculate.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-return-assign */
+/* eslint-disable consistent-return */
```

Comment:
Please, you don't need to disable these rules. Note that the `calculate` function has to return an object to be used to set new state hence let's make the return value consistent.